docs/user/authentication.rst

```diff
@@ -44,6 +44,9 @@ set with `headers=`.
 If credentials for the hostname are found, the request is sent with HTTP Basic
 Auth.
 
+Requests will search for the netrc file at `~/.netrc`, `~/_netrc`, or at the path
```

Comment:
This doesn't list the Windows paths or how to disable this altogether.

Reply:
@sigmavirus24 please check updated commit.